HW3/task1.py

```diff
@@ -0,0 +1,11 @@
+#Задайте список из нескольких чисел. Напишите программу, которая найдёт сумму элементов списка, стоящих на нечётной позиции.
+#Пример:
+#- [2, 3, 5, 9, 3] -> на нечётных позициях элементы 3 и 9, ответ: 12
+
+
+numbers = [2, 3, 5, 9, 3]
+sum = 0
+for count, value in enumerate(numbers):
+    if count % 2:
+            sum += value
```

Comment:
```suggestion
    if count % 2:
        sum += value
```
Один уровень отступа равен четырем пробельными символами у вас гораздо больше.